api/src/main/java/gov/va/vro/api/resources/VroResource.java

```diff
@@ -105,6 +105,41 @@ ResponseEntity<GeneratePdfResponse> generatePdf(
   ResponseEntity<Object> fetchPdf(@PathVariable String claimSubmissionId)
       throws MethodArgumentNotValidException, ClaimProcessingException;
 
+  @Operation(
```

Comment:
I do not think we need a full new end point. Can we add a query param to the existing end point? You also need a "produces" here (take a look at the GET) or in the older post if you add query params.  If you are going to keep this end point can you change  it to a name instead of verb. Do not know what. "immediate-evidence-pdf" maybe?

Reply:
Kept the extra route since it would complicate the `generatePdf` java function and have to parse the 2 different types of responses. Renamed it to `immediate-evidence-pdf`